client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -117,12 +110,12 @@ const NotificationSubscriptions = createReactClass( {
 								{ this.props.translate( 'Default email delivery' ) }
 							</FormLabel>
 							<FormSelect
-								disabled={ this.getDisabledState() }
+								disabled={ this.props.getDisabledState() }
```

Comment:
This looks like a mistake, `getDisabledState` is actually a method and not a prop:
```suggestion
								disabled={ this.getDisabledState() }
```

It occurs in multiple locations in this file.

Reply:
`getDisabledState` is a method from the `withFormBase` hoc which passes it down as a prop to `NotificationSubscriptions` so that should be correct.

Reply:
🤦 of course.